pyorthanc/instance.py

```diff
@@ -165,6 +168,16 @@ def simplified_tags(self) -> Dict:
         """
         return dict(self.client.get_instances_id_tags(self.id_, params={'simplify': True}))
 
+    @property
+    def labels(self) -> List[str]:
+        return self.get_main_information()['Labels']
```

Comment:
That is a good point. I did this because `self.get_main_information()` acts as a cache, but I think it is a bad design since users have to force a refresh to get the updated data.

I plan to change the resources inside (Patient/Study/Series/Instance) in a subsequent PR.